docs/reference/mouse_bindings.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Mouse bindings
+---
+
+The mousemap files (which have the extension `.sublime-mousemap`)
+control what commands are executed when a user performs an action
+with a mouse e.g. clicking a mouse button, scrolling the scroll
+wheel etc.
+
+
+## File Format
+
+Here is a small excerpt from `Default/Default (Windows).sublime-mousemap`.
+
+```json
+[
+	// Basic drag select
+	{
+		"button": "button1", "count": 1,
+		"press_command": "drag_select"
+	},
+	{
+		"button": "button1", "count": 1, "modifiers": ["ctrl"],
+		"press_command": "drag_select",
+		"press_args": {"additive": true}
+	},
+	{
+		"button": "button1", "count": 1, "modifiers": ["alt"],
+		"press_command": "drag_select",
+		"press_args": {"subtractive": true}
+	},
+]
+```
+
+The following are the keys that a single JSON entry 
+in a mousemap file can take.
+
+`button`
+: The name of the button.
+
+  This defines the name of the button.
+  E.g. `button1` defines the left mouse button, `button2`
+  defines the right mouse button etc.
+  Similarly, if a command has to be executed when the scroll
+  wheel scrolls, then we have `scroll_up` & `scroll_down`.
+
+`modifiers`
+: A list of modifier keys.
+
+  This defines a list of modifiers keys that have to 
+  be pressed simultaneously (along with the corresponding button click)
+  for the command to be triggered. E.g. `["ctrl", "alt"]`.
```

Comment:
A list of supported modifiers or a reference to the key bindings page, which should have that already, would be appreciated.